docs/api/session.md

```diff
@@ -1330,18 +1330,35 @@ the initial state will be `interrupted`. The download will start only when the
 
 Returns `Promise<void>` - resolves when the session’s HTTP authentication cache has been cleared.
 
-#### `ses.setPreloads(preloads)`
+#### `ses.setPreloads(preloads)` _Deprecated_
 
 * `preloads` string[] - An array of absolute path to preload scripts
 
 Adds scripts that will be executed on ALL web contents that are associated with
 this session just before normal `preload` scripts run.
 
-#### `ses.getPreloads()`
+#### `ses.getPreloads()` _Deprecated_
 
 Returns `string[]` an array of paths to preload scripts that have been
 registered.
```

Comment:
We should clarify that this will only return webContents preload scripts, and not service worker preloads.

Reply:
added deprecation notes in [4bf498d](https://github.com/electron/electron/pull/44411/commits/4bf498de18a8452bb86c84fc582edebd1d7158e9), good call out!